src/components/FileDownload/FileDownload.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import cx from 'classnames';
+
+type FileDownloadProps = {
+  className?: string;
+  href: string;
+  label?: string;
+  name?: string;
+  size?: string;
+  type?: string;
+};
+
+export const FileDownload = ({
+  className,
+  href,
+  label = 'Download',
+  name,
+  size,
+  type
+}: FileDownloadProps) => {
+  const classNames = cx('cc-file-download', {
+    [`${className}`]: className
```

Comment:
Template literal/placeholder not actually needed in this instance - you can initialise as just `[className]` and all still works - much of a muchness though 🙃

Reply:
Yeah copied and pasted for the umpteenth time 😬 

Removed it here but I'll do all of the other 29 instances in a separate PR 😆